src/ppp_connection_manager.sh

```diff
@@ -4,10 +4,12 @@ source functions.sh
 
 for i in {1..4}; do
     bash configure_modem.sh
+    MODEM_CONFIG=$?
     
     if [[ $MODEM_CONFIG -eq 0 ]]; then
         break
     fi
+    i=$(($i+1))
```

Comment:
Hello,
Thank you for your contribution.
But this line is not necessary here. Because, `for I in {1..4}` line provides auto incrementation for `i`  already. If there is not something that I misunderstand I'll close this PR after your response.

Reply:
indeed - sorry! as $MODEM_CONFIG was never set, the script exited early - no issue on this line